sections/03_categorize.md

```diff
@@ -290,80 +290,95 @@ by allowing more efficient use of the costly practice of manual annotation.
-would still encounter technological and legal barriers. A responsibility to
-protect patient privacy limits the ability to  openly share large patient
-datasets. As described above, labeled data are already expensive to obtain. Even
-after they are generated restrictions on sharing can hamper their broad
-distribution. All of these factors combine to result in small samples sizes that
-restrict the number of parameters that can be trained in a model. Though the
-lack of sharing may also hamper reproducibility and physician confidence in
-results, recently described techniques such as Continuous Analysis
-[@doi:10.1038/nbt.3780] can allow reproducible computing on private data. Using
-such techniques intermediate results can be automatically tracked and shared
-without sharing the original data, which may help to address concerns around
-physician confidence.
-
-Raw data isn't the only point of concern in the domain with regards to privacy.
+EHRs are designed chiefly for clinical, administrative and financial purposes, such as patient care, insurance and  billing [@doi:10.1038/nrg3208]. Research is at best a tertiary priority, presenting  significant challenges to EHR-based research in general, and particularly to data intensive deep learning research. These difficulties can be grouped into three areas: local bias, wider standards and legal issues. Note these problems are not restricted to EHR but can also apply to any large biomedical dataset, e.g. clinical trial data.
+
+Even within the same healthcare system, EHRs can be used differently
+[@pmid:24159271 @pmid:21347133]. Individual users have unique usage patterns,
+with different departments and different hospitals having different priorities
+which code patients and introduce missing data in a non-random fashion
+[@pmid:12849909]. Patient data may be kept across several "silos" within a
+single health system. Even the most basic task of matching patients across
+systems can be challenging due to data entry issues [@pmid:27134610]. The
+situation is further exacerbated by the ongoing introduction, evolution and
+migration of EHR systems, especially where reorganized and acquired healthcare
+facilities have to merge. As a result, EHR can be less complete and less
+objective than expected.
+
+In the wider picture, standards for EHRs are many and evolving. Proprietary
+systems, indifferent and scattered use of health information standards and
+controlled terminologies makes combining and comparison of data across systems
+challenging [@url:http://www.ncrr.nih.gov/
+publications/informatics/ehr.pdf][@doi:10.1016/j.jbi.2014.10.006]. Further
+diversity arises from variation in languages, healthcare practices and
+demographics. Merging EHR gathered in different systems (and even under
+different assumptions) is challenging [@isbn:9783319448398].
+
+Combining or replicating studies across systems thus requires controlling for
```

Comment:
@agapow we don't currently have support for ISBNs.  Can you please replace this reference with `[@doi:10.1007/978-3-319-44839-8]` and confirm that DOI is for the book you intended?